CONTRIBUTING.rst

```diff
@@ -387,37 +434,43 @@ checkout of MLflow:
       Generating ~/mlflow/mlflow/store/db_migrations/versions/b446d3984cfa_add_new_field_to_db.py
 
 
-These commands generate a new migration script (e.g. at
-``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``) that you should then edit to add
-migration logic.
-
+These commands generate a new migration script (e.g., at ``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``)
+that you should then edit to add migration logic.
 
-Launching the Development UI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-We recommend `Running the Javascript Dev Server`_ - otherwise, the tracking frontend will request
-files in the ``mlflow/server/js/build`` directory, which is not checked into Git.
-Alternatively, you can generate the necessary files in ``mlflow/server/js/build`` as described in
-`Building a Distributable Artifact`_.
+Writing MLflow Examples
++++++++++++++++++++++++
+The ``mlflow/examples`` directory has a collection of quickstart tutorials and various simple examples that depict MLflow tracking,
+project, model flavors, model registry, and serving use cases. Written in `PEP8 Python coding style <https://www.python.org/dev/peps/pep-0008/>`_, these
```

Comment:
Maybe we can remove this reference to PEP8 coding style, as @harupy's linting work now ensures that we lint our examples

Reply:
cool, will do!